enterprise/backend/test/metabase_enterprise/sso/integrations/jwt_test.clj

```diff
@@ -77,7 +77,16 @@
                                                          {:request-options {:redirect-strategy :none}}
                                                          :redirect default-redirect-uri)
             redirect-url (get-in result [:headers "Location"])]
-        (is (str/starts-with? redirect-url default-idp-uri))))))
+        (is (str/starts-with? redirect-url default-idp-uri)))))
+  (testing (str "JWT configured with a redirect-uri containing query params, "
+                "a GET request should result in a redirect to the IdP as a correctly formatted URL")
```

Comment:
nit: often when we add test cases for bug fixes we add a mention of the issue number, might be worth doing here

Reply:
Ah, thanks. Do we add it in a comment or in a string somewhere inside the test?

Reply:
just in the testing context string, like https://github.com/metabase/metabase/pull/19530/files#diff-c00373b640a0d87270c78f3b00299b78d569505bb3cfab808d7c039ec7a836deR473-R474